### ISSUE TITLE
Remove text limits

### ISSUE THREAD

**Issue by TecFuture** [ScratchAddons/ScratchAddons#3342]
### Is your request related to a problem? Please describe.

In scratch you can put certain **command's** like "random" and "all" (**in** list blocks) and sometimes its _needed_ in **other situations** and blocks
### Describe the solution you'd like

The text limits in scratch blocks to be completely removed in all blocks, such as only number text input.
### Describe alternatives you've considered
You can bypass scratch text limits by copy pasting, but this isn't efficient.

### Additional context
This a example of an item block with text
![block_8_21_2021-12_03_26 PM](https://user-images.githubusercontent.com/82056497/130327868-30c0dd4a-3d22-40ca-95a1-8642ad1cc33e.png)

**Comment by towerofnix** [comment]
Another option would be to automatically insert the "item (random) of (list)" and "delete (random) of (list)" blocks to the block palette—so, a copy of the default block, but with the input automatically filled out so you can drag and use it just like that. This lets you avoid typos that could come up typing in those inputs, while still giving full access to the blocks.

(I don't think "delete (all) of (list)" needs to be inserted into the palette because we already have the new 3.0 block "delete all of (list)".)

**Comment by TecFuture** [comment]
> Another option would be to automatically insert the "item (random) of (list)" and "delete (random) of (list)" blocks to the block palette—so, a copy of the default block, but with the input automatically filled out so you can drag and use it just like that. This lets you avoid typos that could come up typing in those inputs, while still giving full access to the blocks.
> 
> (I don't think "delete (all) of (list)" needs to be inserted into the palette because we already have the new 3.0 block "delete all of (list)".)

Yes, but you can use it in another situations that's why I suggest removing ALL text limits

**Comment by mxmou** [comment]
> Yes, but you can use it in another situations that's why I suggest removing ALL text limits

Are there any other blocks that expect numbers but also accept text?

**Comment by TecFuture** [comment]
> > Yes, but you can use it in another situations that's why I suggest removing ALL text limits
> 
> Are there any other blocks that expect numbers but also accept text?

I just checked, there aren't but putting in the palette isn't the best option since nearly ALL lists contain the word item and expect item # input after that.

**Comment by apple502j** [comment]
`toListIndex` supports `last` and `random` for all casting, and `all` when it makes sense.

**Comment by TecFuture** [comment]
> `toListIndex` supports `last` and `random` for all casting, and `all` when it makes sense.

Yea so. remove text limits OR dropdowns.
